### ISSUE TITLE
Refactor mono/stereo

### ISSUE THREAD

**Issue by mickvangelderen** [mickvangelderen/vr-lab#12]
This duplication is bothering. 

https://github.com/mickvangelderen/vr-lab/blob/wip/renderer/src/main.rs#L854-L1178

For monoscopic rendering there is no pose (camera base/character to head) or head_to_eye transforms. We could refactor the code to return identity matrices and an array of 1 eye for the mono case so we can do all the now duplicated stuff only once.

**Comment by mickvangelderen** [comment]
No longer relevant
